scripts/release.Dockerfile

```diff
@@ -51,6 +51,3 @@ RUN git clone https://github.com/Homebrew/brew /home/linuxbrew/.linuxbrew
 ENV PATH=/home/linuxbrew/.linuxbrew/bin:$PATH
 
 RUN mkdir -p ~/.windmill
-
-ENTRYPOINT ["goreleaser"]
```

Comment:
what's with this change? i don't think we want to use the upstream entrypoint

Reply:
This was a fun adventure:
* Toast does not have support for overriding entrypoint (it expects to be dropped into some sort of shell)
* You CAN pass arbitrary flags, so I tried to hack around with `--entrypoint=""`
  * Toast uses `--init` flag with Docker to get a Tini init to avoid orphaned processes since not all shells handle this
  * Tini complains it can't exec `""` 😬 

The upstream `ENTRYPOINT` is `bash` and there's no `CMD`.

It's irrelevant to CircleCI since they override the entrypoint (presumably due to situations like this, which I've had to battle before with GitLab CI 😓)

Reply:
ya, i think this just speaks to my earthly comment haha